<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
   <layout class="QGridLayout" name="m_mainWndGridLayout">
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>26</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="locale">
    <locale language="English" country="UnitedKingdom"/>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>369</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>524287</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2" rowspan="4">
      <widget class="QTreeView" name="treeView_hierarchy">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>130</height>
        </size>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="title">
        <string>Camera</string>
       </property>
       <widget class="QPushButton" name="btn_resetCamera">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>311</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotate: Alt+LMB
Translate: Alt+RMB
Zoom in/out: Wheel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0" rowspan="4">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Transform</string>
       </property>
       <widget class="QDoubleSpinBox" name="dsb_tX">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>16</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>16</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>y</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>80</y>
          <width>16</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>z</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_tZ">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>110</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_tY">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Translation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Rotation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_rX">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>160</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_rY">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_rZ">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>160</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_sX">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>210</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_sY">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_sZ">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>210</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>29</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QRadioButton" name="rb_initT">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Initial</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">btnG_TMode</string>
         </attribute>
        </widget>
        <widget class="QRadioButton" name="rb_curT">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Current</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">btnG_TMode</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="btn_addBone">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_deleteBone">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btnG_TMode"/>
 </buttongroups>
</ui>
